src/main/webapp/WEB-INF/jsp/budget/BudgetDistributionAndIncome.jsp

```diff
@@ -13,50 +13,61 @@
+
 <kra-b:swapProposalDevelopmentEditModes/>
 
-	
+
 <kra-b:swapProposalDevelopmentEditModes/>
 <c:set var="readOnly" value="${(not KualiForm.editingMode['modifyBudgets']) && ( not parentReadOnlyFlag )}" scope="request" />
 <c:set var="viewOnly" value="${readOnly}" scope="request" />
 <bean:define id="proposalBudgetFlag" name="KualiForm" property="document.proposalBudgetFlag"/>
 
 <kul:documentPage
-	showDocumentInfo="true"
-	htmlFormAction="${KualiForm.actionPrefix}DistributionAndIncome"
-	documentTypeName="${KualiForm.docTypeName}"
-  	headerDispatch="${KualiForm.headerDispatch}"
-  	headerTabActive="distributionAndIncome"
-  	extraTopButtons="${KualiForm.extraTopButtons}"
-  	showTabButtons="true">
-  	
-	 <c:choose>
-		 <c:when test="${proposalBudgetFlag}">
-        	<div align="right"><kul:help parameterNamespace="KC-B" parameterDetailType="Document" parameterName="budgetDistributionAndIncomeHelp" altText="help"/></div>
-         </c:when>
-         <c:otherwise>
-        	<div align="right"><kul:help parameterNamespace="KC-AB" parameterDetailType="Document" parameterName="awardBudgetDistributionAndIncomeHelpUrl" altText="help"/></div>
-        </c:otherwise>
-    </c:choose>
-	
+		showDocumentInfo="true"
+		htmlFormAction="${KualiForm.actionPrefix}DistributionAndIncome"
+		documentTypeName="${KualiForm.docTypeName}"
+		headerDispatch="${KualiForm.headerDispatch}"
+		headerTabActive="distributionAndIncome"
+		extraTopButtons="${KualiForm.extraTopButtons}"
+		showTabButtons="true">
+
+	<c:choose>
+		<c:when test="${proposalBudgetFlag}">
+			<div align="right"><kul:help parameterNamespace="KC-B" parameterDetailType="Document" parameterName="budgetDistributionAndIncomeHelp" altText="help"/></div>
+		</c:when>
+		<c:otherwise>
+			<div align="right"><kul:help parameterNamespace="KC-AB" parameterDetailType="Document" parameterName="awardBudgetDistributionAndIncomeHelpUrl" altText="help"/></div>
+		</c:otherwise>
+	</c:choose>
+
 	<div align="center">
 		<kra-b:budgetCostSharing />
 		<kra-b:budgetUnrecoveredFandA />
 		<kra-b:budgetProjectIncome />
 		<kul:panelFooter />
 	</div>
 
-	<kul:documentControls 
-		transactionalDocument="false"
-		suppressRoutingControls="true"
-		extraButtonSource="${extraButtonSource}"
-		extraButtonProperty="${extraButtonProperty}"
-		extraButtonAlt="${extraButtonAlt}"
-		viewOnly="${readOnly}"
-		suppressCancelButton="true"
-		/>
+	<kul:documentControls
+			transactionalDocument="false"
+			suppressRoutingControls="true"
+			extraButtonSource="${extraButtonSource}"
+			extraButtonProperty="${extraButtonProperty}"
+			extraButtonAlt="${extraButtonAlt}"
+			viewOnly="${readOnly}"
+			suppressCancelButton="true"
+	/>
 
+	<SCRIPT type="text/javascript">
+		var kualiForm = document.forms['KualiForm'];
+		var kualiElements = kualiForm.elements;
+		var $j = jQuery.noConflict();
+	</SCRIPT>
+	<script language="javascript" src="scripts/kuali_application.js"></script>
```

Comment:
kuali_application.js and ua_kuali_application.js are already included in all pages, no need to specifically re-add on this .jsp.